backend/src/main/java/com/cic/service/MatchingService.java

```diff
@@ -10,18 +10,34 @@
 public class MatchingService {
 
   private final MatchingAlgorithm matcher;
+  private final float acceptedThreshold;
 
-  public MatchingService(MatchingAlgorithm matcher) {
+  public MatchingService(final MatchingAlgorithm matcher, final float acceptedThreshold) {
     this.matcher = matcher;
+    this.acceptedThreshold = acceptedThreshold;
   }
 
-  public List<Match> returnMatches(Set<Person> mentees, Set<Person> mentors) {
-    List<Match> matches = new ArrayList<>();
-
-    // for (int i = 0; i < mentees.length; i++) {
-    // matches.add(new Match().mentor(mentees[i]).mentee(mentors[i]));
-    // }
-
+  public List<Match> returnMatches(final Set<Person> mentees, final Set<Person> mentors) {
+    final List<Match> matches = new ArrayList<>();
+    final Set<Person> tempMentors = mentors;

```

Comment:
Is `tempMentors` a good name for this variable? Would something like `mentorsToFindAMatchFrom` or `mentorsAvailableForMatching` or something more descriptive be better?

Reply:
done